src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I did not tell you to not understand, study, or test the code of this PR, but if you want to get me wrong, I'll hardly be able to keep you from doing that. "Agreeing" while being offended shows that you completely disagree with me, so please let me know why you disagree. If you make this PR depend on the success of other patches, there has to be some concern regarding the changes by this patch. Please verbalize these concerns (and not the concerns about the LMR code in general), so that I can understand them.